bld/namelist_files/namelist_defaults_ctsm_tools.xml

```diff
@@ -55,48 +55,19 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
-<scripgriddata hgrid="5x5min"   lmask="ISRIC-WISE"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_ISRIC-WISE_c111114.nc</scripgriddata>
-<scripgriddata hgrid="5x5min"   lmask="ORNL-Soil"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_ORNL-Soil_c170630.nc</scripgriddata>
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_nomask_fromIGBP_c191107.nc</scripgriddata>
 <scripgriddata hgrid="10x10min" lmask="nomask"    
    >lnd/clm2/mappingdata/grids/SCRIPgrid_10x10min_nomask_c110228.nc</scripgriddata>
-<scripgriddata hgrid="10x10min" lmask="IGBPmergeICESatGIS"    
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_10x10min_IGBPmergeICESatGIS_c110818.nc</scripgriddata>
-<scripgriddata hgrid="3x3min"    lmask="MODIS"    
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3minx3min_MODIS_c110915.nc</scripgriddata>
-<scripgriddata hgrid="3x3min"    lmask="MODIS-wCsp"    
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3minx3min_MODISwcspsea_c151020.nc</scripgriddata>
-<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="MODIS"     >64bit_offset</scripgriddata_lrgfile_needed>
-<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="MODIS-wCsp">64bit_offset</scripgriddata_lrgfile_needed>
-<scripgriddata hgrid="3x3min"    lmask="USGS"     
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3x3_USGS_c120912.nc</scripgriddata>
-<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="USGS">64bit_offset</scripgriddata_lrgfile_needed>
-<scripgriddata hgrid="3x3min"    lmask="LandScan2004"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3minx3min_LandScan2004_c120517.nc</scripgriddata>
-<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="LandScan2004">64bit_offset</scripgriddata_lrgfile_needed>
-<scripgriddata hgrid="3x3min"    lmask="GLOBE-Gardner"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3minx3min_GLOBE-Gardner_c120922.nc</scripgriddata>
-<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="GLOBE-Gardner">64bit_offset</scripgriddata_lrgfile_needed>
-<scripgriddata hgrid="3x3min"    lmask="GLOBE-Gardner-mergeGIS"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3minx3min_GLOBE-Gardner-mergeGIS_c120922.nc</scripgriddata>
-<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="GLOBE-Gardner-mergeGIS">64bit_offset</scripgriddata_lrgfile_needed>
-<scripgriddata hgrid="360x720cru"   lmask="cruncep" 
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_360x720_cruncep_c120830.nc</scripgriddata>
+<scripgriddata hgrid="3x3min"    lmask="nomask"    
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_3x3min_nomask_fromGLOBE_c191105.nc</scripgriddata>
+<scripgriddata_lrgfile_needed hgrid="3x3min" lmask="nomask">64bit_offset</scripgriddata_lrgfile_needed>
 <scripgriddata hgrid="360x720cru"   lmask="nomask" 
    >lnd/clm2/mappingdata/grids/SCRIPgrid_360x720_nomask_c120830.nc</scripgriddata>
-<scripgriddata hgrid="0.9x1.25"  lmask="GRDC"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.9x1.25_GRDC_c130307.nc</scripgriddata>
+<scripgriddata hgrid="0.9x1.25"  lmask="nomask"
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.9x1.25_nomask_c191014.nc</scripgriddata>
```

Comment:
Regarding the diffs in VIC variables mentioned in 57e4b24,
I have confirmed that the two files above differ only in that the mask = 1 everywhere in the nomask file.